src/components/DailyRoutinePage/DailyRoutine.js

```diff
@@ -1,35 +1,84 @@
 import "./DailyRoutine.css";
 import AddedProductCard from "./AddedProductCard.js";
 import { FaArrowLeft } from "react-icons/fa";
+import { useParams, useHistory } from "react-router-dom";
+import { getDataFromLocalStorage } from "../../utility/localStorage.js";
+import { useEffect, useState } from "react";
 
 export default function DailyRoutine() {
+  const { weekday } = useParams();
+  const history = useHistory();
+  const [allItems, setAllItems] = useState([]);
```

Comment:
`allItems` is a rather generic name, could you rename it to a more descriptive name?